src/js/base/model.js

```diff
@@ -84,6 +85,12 @@ export default Backbone.Model.extend(extend({
     return isFunction(messages[category]) ? messages[category] : this[messages[category]];
   },
   handleMessage({ category, payload }) {
+    if (payload.attributes) {
+      payload.attributes.updated_at = dayjs.utc().format();
+    } else {
+      payload.attributes = { updated_at: dayjs.utc().format() };
+    }
```

Comment:
Probably a better way to do this. But just attaching `attributes.updated_at` to the payload data before its passed to the individual entity services.

Reply:
`payload.attributes = extend(payload.attributes, { updated_at: day...`